lib/routemaster/cache.rb

```diff
@@ -94,17 +92,7 @@ def bust(url)
     # header.
     #
     # @return [Response], which responds to `status`, `headers`, and `body`.
-    def get(url, version:nil, locale:nil)
-      key   = "cache:#{url}"
-      field = "v:#{version},l:#{locale}"
-
-      # check cache
-      if payload = @redis.hget(key, field)
-        publish(:cache_hit, url)
-        return Response.new(JSON.parse(payload))
-      end
-
-      # fetch data
+    def get(url, version: nil, locale: nil)
```

Comment:
Just so I understand: I guess you'd expand the API of this (`Cache`), to support POST, PATCH, DELETE requests? As it'd effectively be your client.

Or do you plan on extending `Fetcher` instead?

Both are now poorly named I suppose.